examples/tv-casting-app/tv-casting-common/src/CastingServer.cpp

```diff
@@ -365,7 +411,7 @@ TargetVideoPlayerInfo * CastingServer::ReadCachedTargetVideoPlayerInfos()
     CHIP_ERROR err = mPersistenceManager.ReadAllVideoPlayers(mCachedTargetVideoPlayerInfo);
     if (err != CHIP_NO_ERROR)
     {
-        ChipLogError(AppServer, "ReadAllVideoPlayers error: %" CHIP_ERROR_FORMAT, err.Format());
+        ChipLogProgress(AppServer, "ReadAllVideoPlayers error: %" CHIP_ERROR_FORMAT, err.Format());
```

Comment:
was this intentional?

Reply:
Yes, because a non CHIP\_NO\_ERROR value here is usually returned just because the cache is empty and that is not an issue.